assets/javascripts/modules/analytics/ga.js

```diff
@@ -64,8 +64,19 @@ define(['modules/analytics/analyticsEnabled',
         ga('jellyfishGA.send', 'pageview');
     }
 
+    function trackEvent(eventLabel) {
+        var upgrading = (eventLabel === 'Rate Plan Change' && guardian.pageInfo.productData.initialProduct !== guardian.pageInfo.productData.productPurchasing) ? 1 : 0;
+        ga('membershipPropertyTracker.send', 'event', {
+            eventCategory: 'Subscriptions Checkout',
+            eventAction:  guardian.pageInfo.productData.productType,
+            eventLabel: eventLabel,
+            dimension11: guardian.pageInfo.productData.productType + ' - ' + guardian.pageInfo.productData.productPurchasing,
```

Comment:
thanks for the update, actually I don't know a lot about JS or GA and I wasn't specifying we should achieve it by sending something different to GA.. for example in scala we might create a writer with the mappings that can be checked separately and then just use the friendly names in the code